floorp/browser/locales/en-US/floorp.ftl

```diff
@@ -33,19 +33,19 @@ multirow-tabs-newtab =
  .label = Place the "Open a new tab" button at the end of the lowest row of tabs
 multirow-tabs-value = Number of rows when multi-row tabs are enabled
 enable-tab-sleep = 
- .label = Enable Tab Sleep
-tab-sleep-timeout-minutes-value = Put inactive tabs to sleep after the following period of time (minutes)
+ .label = Enable Sleeping Tabs
+tab-sleep-timeout-minutes-value = Tabs will sleep after being inactive for (minutes)
```

Comment:
```suggestion
tab-sleep-timeout-minutes-value = Tabs will be put into sleep after being inactive for (minutes)
```

Reply:
'Tabs will sleep' sounds more natural to me than 'Tabs will be put into sleep', given the use of 'sleep' as the term for it.

I changed it to that in this merge request as I realised after the prior one that "put to sleep" sounds too much like an action with a seriously morbid meaning to it.